packages/module/src/ServiceCard/ServiceCard.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Button, ButtonVariant, Card, CardBody, CardFooter, CardHeader, Text, TextContent, TextVariants } from '@patternfly/react-core';
+import { HelperText } from '@patternfly/react-core/dist/dynamic/components/HelperText';
+import { HelperTextItem } from '@patternfly/react-core/dist/dynamic/components/HelperText';
+import { createUseStyles } from 'react-jss';
+
+export interface ServiceCardProps {
+  /** Title for card */
+  title: string;
+  /** Subtitle for card */
+  subtitle: string;
+  /** Custom description */
+  description: string;
+  /** icon for service card */
+  icon: React.ReactNode;
+  /** Whether to show if button is disabled */
+  showDisabledButton?: boolean;
+  /** Helper text for card */
+  helperText?: string;
+  /** learn more button url*/
+  learnMoreUrl: string;
+  /** Optional launch button url*/
+  launchUrl?: string;
+  /** Optional foot to override default Learn More and Launch buttons */
+  footer?: React.ReactElement
+  /** Optional custom OUIA ID */
+  ouiaId?: string | number;
+}
+
+const useStyles = createUseStyles({
+  card: {
+    height: ('var(--pf-v5-u-h-100)')
+  },
+  image: {
+    marginRight: ('var(--pf-v5-global--spacer--md)'),
+    width: 48
+  },
+  launchButton: {
+    marginRight: ('var(--pf-v5-global--spacer--sm)')
```

Comment:
```suggestion
    marginRight: 'var(--pf-v5-global--spacer--md)'
```
I think this one should be `md`